source/solvers/vof.cc

```diff
@@ -364,6 +413,41 @@ VOF<dim>::postprocess(bool first_iteration)
           this->pcout << "L2 error phase : " << phase_error << std::endl;
         }
     }
+
+  //  bool conservation_monitoring(true);
+  //  int  fluid_index(1);
+
+  if (simulation_parameters.multiphysics.conservation_monitoring)
+    {
+      double volume =
+        calculate_volume(simulation_parameters.multiphysics.fluid_index);
+
+      auto         mpi_communicator = triangulation->get_communicator();
+      unsigned int this_mpi_process(
+        Utilities::MPI::this_mpi_process(mpi_communicator));
+
+      if (this_mpi_process == 0)
+        {
+          // Set conservation monitoring table
+          if (simulation_control->is_steady())
+            {
+              error_table_fs.add_value(
+                "cells", this->triangulation->n_global_active_cells());
+            }
+          else
+            {
+              error_table_fs.add_value("time",
+                                       simulation_control->get_current_time());
+            }
+          error_table_fs.add_value("fluid_volume", volume);
+          error_table_fs.set_scientific("fluid_volume", true);
```

Comment:
See above comments on keeping a seperate table for vvolume conservation. You don't need an error table here you can use a regular table (and even that is only required if you want to print the entire volume conservation history, which you could want to :) ).

Reply:
maybe it won't be relevant in the long run, but for now yes I wanted to have the entire history!

Changed, thank you for the review :)